app/controllers/concerns/api/v3/sync_to_user.rb

```diff
@@ -1,20 +1,22 @@
 module Api::V3::SyncToUser
   extend ActiveSupport::Concern
+
   included do
```

Comment:
Instead of an additional mixin called `PrioritisableByFacility` (or `PrioritisableByPatient` it should've been called now), we stick the default and most common behavior in `SyncToUser` for `current_facility_records`, `other_facility_records`.

Certain controllers like Facilities and Protocols can override it to avoid syncing scoped by `region_records` since we sync all of them all the time.

Notice that we've also changed the scope to be `patient` here instead of `facility`; this is more in-line with a proper patient-centric sync.